Benchmarks/P90AbsoluteThresholds/P90AbsoluteThresholds.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) 2023 Ordo One AB
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+
+import Benchmark
+import DateTime
+
+let benchmarks = {
+    var testName: String = "testName should be set"
```

Comment:
Making `testName` optional would reduce readability I guess but this looks a bit weird.

What about `var testName: String!`?